client/src/ui/components/hyperstructures/HyperstructurePanel.tsx

```
@@ -173,9 +189,34 @@
+                    <Select
+                      onValueChange={(access: keyof typeof Access) => {
+                        setAccess(BigInt(Access[access]));
+                      }}
+                    >
+                      <SelectTrigger className="w-[140px] text-gold h-10 text-lg">
+                        <SelectValue placeholder={separateCamelCase(hyperstructure.access)} />
+                      </SelectTrigger>
+                      <SelectContent className="bg-black/90 text-gold">
+                        {Object.keys(Access)
+                          .filter((access) => {
+                            if (!isNaN(Number(access))) return false;
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)